Entities/Common/Building/PlacementCommon.as

```diff
@@ -182,6 +183,14 @@ bool isBuildableAtPos(CBlob@ this, Vec2f p, TileType buildTile, CBlob @blob, boo
 				}
 			}
 		}
+
+
+		if (isSeed)
+		{
+			// from canGrow.as
+			return (map.isTileGroundStuff(map.getTile(p + Vec2f(0, 8)).type));
```

Comment:
This includes dirt back wall. I would narrow it down so you can only place it on dirt.

Reply:
Should be fixed in latest commit